app/src/main/java/org/simple/clinic/summary/teleconsultation/api/TeleconsultPhoneNumber.kt

```diff
@@ -1,10 +1,13 @@
 package org.simple.clinic.summary.teleconsultation.api
 
+import android.os.Parcelable
 import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
+import kotlinx.android.parcel.Parcelize
 
+@Parcelize
 @JsonClass(generateAdapter = true)
 data class TeleconsultPhoneNumber(
     @Json(name = "phone_number")
     val phoneNumber: String
-)
+) : Parcelable
```

Comment:
Was this change to `Parcelable` meant to be a part of the commit where we pass this as an argument to the sheet?

Reply:
Yes, moved it